addons/website/static/src/scss/website.scss

```diff
@@ -190,6 +190,16 @@ $-seen-urls: ();
     }
 };
 
+html {
+    // In the line below, a white background is set on HTML due to the fallback
+    // iframe used in edit mode. This second iframe is placed behind the website
```

Comment:
"edit mode" is confusing: the iframe is not at all important for "edit mode", just for navigation in the backend (outside of edit mode).